### ISSUE TITLE
Migrate existing WordPress dictionaries into the Cloud/Mongo database

### ISSUE THREAD

**Issue by johntseng** [sillsdev/webonary#333]
There may be some dictionaries that are only in WordPress database, and we cannot ask users to re-upload them to the Cloud/Mongo database.  It would be nice to migrate all those databases.

Han has a POC script for this.

**Comment by hanatgit** [comment]
Yes, this was used during initial development, but will need to be touched up to make sure it will work cleanly with these older dictionaries, especially in taking account relevant fields in the xhtml, https://github.com/sillsdev/webonary/blob/develop/webonary-cloud-api/tools/post-legacy.ts
